app/views/carts/show.html.haml

```diff
@@ -21,5 +21,6 @@ In cart:
   $
 - if  @cart.shopping_cart_items.any?
   = link_to 'Clear cart', cart_clear_cart_path(@cart, restaurant_id: @restaurant.id), class: 'btn btn-primary'
+= link_to 'Pay with Card',
```

Comment:
You probably already know that you're not linking to a path here, but better safe than sorry :)

Reply:
Thanks I know I just put it there and then I was going to add a path, then I just deleted this...